test/biophysics.jl

```diff
@@ -89,26 +96,29 @@ Q_in = Q_solar + Q_IR_in + Q_metab
 Q_out = Q_IR_out + Q_conv + Q_evap + Q_resp + Q_cond
 Q_in - Q_out
 
-T_x = T_air
 
 T_surf_s = find_zero(heat_balance, (T_air - 40K, T_air + 100K), Bisection())
 T_surf_C = (Unitful.ustrip(T_surf_s) - 273.15)°C
 
 
 # using structs to pass parameters
 
+# define the geometry
 mass_organism = 0.04kg
 ρ_organism = 1000kg/m^3
 shapeb_organism = 2
 shape_organism = Cylinder(mass_organism, ρ_organism, shapeb_organism)
 body_organism = Body(shape_organism, Naked())
 
+# construct the Model which holds the parameters of the organism in the Organism concrete struct, of type AbstractOrganism
 lizard = Model(Organism(body_organism, OrganismParams()))
+# get the environmental parameters
 environmental_params = EnvironmentalParams()
+# get the variables for both the organism and environment
 variables = (organism=OrganismalVars(), environment=EnvironmentalVars())
 
-heat_balance(lizard, environmental_params, variables)
-
-T_air = EnvironmentalVars().Ta
+# define the method 'heat_balance' for passing to find_zero, which dispatches off 'lizard' 
+T_air = EnvironmentalVars().T_air
+heat_balance(T_air, lizard, environmental_params, variables)
```

Comment:
This line is giving the following error: UndefVarError: stripparams not defined

Reply:
Yes, I got that one :) You have to uncomment the method of `heat_balance()` in line 10. That method dispatches on insulation type after parameters have been stripped

Reply:
Ah, and after managing to use the appropriate method, Unitful throws an error about a dimensional issue that tracks back to get_Tsurf_Tlung (line 511). I think you might want to strip units there because its adding K to W m⁻¹

Reply:
I was thinking this was part of my issues before with ModelParameters but all works now that I've updated the package and I've put the units back in.